src/libraries/System.Text.RegularExpressions/tests/UnitTests/SymbolicRegexTests.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Globalization;
+using Xunit;
+using System.Text.RegularExpressions.Symbolic;
+using System.Collections.Generic;
+
+namespace System.Text.RegularExpressions.Tests
+{
+    public class SymbolicRegexTests
+    {
+        [Theory]
+        [InlineData(UnicodeCategory.DecimalDigitNumber, 370)] //37 different kinds of decimal digits
+        [InlineData(UnicodeCategory.Surrogate, 2048)]         //1024 low surrogates and 1024 high surrogates
+        public void BDDCardinalityTests(UnicodeCategory category, uint expectedCardinality)
```

Comment:
If "Cardinality" is meant to mean "NumberOfRepresentedCharacters", can we say that instead? :)